css/editor.css

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
We don't push asset build files such as `.css`, `.min.js` or `.map` files as these are built before deployment.